src/annotator/util/frame-util.js

```diff
@@ -43,6 +43,16 @@ function isValid (iframe) {
   return iframe.className !== 'h-sidebar-iframe';
 }
 
+function isReady (iframe, callback) {
```

Comment:
I think isLoaded is probably the least confusing of the two. `isReady` is the one that could mean anything, really. So the question we should answer with the name is, is ready for what? With that, perhaps `isDocumentReady` or `isContentWindowReady` something like that would move the needle for understanding